src/preferences/AllPreferences.js

```diff
@@ -31,20 +31,48 @@
  * PreferencesManager.on("change", AllPreferences.EMMET, preferenceChanged);
    preferenceChanged();
  * ```
+ * NB: JSDOC/Comments marked with * are required for adding the option to Menu
  */
 
-   define(function (require, exports, module) {
+define(function (require, exports, module) {
     const PreferencesManager = require("preferences/PreferencesManager");
     const Strings = require("strings");
+    const Menus = require("command/Menus");
```

Comment:
move this command registration to `DocumentCommandHandlers.js` and menu addition to `DefaultMenus.js`. thats where we usually add shared code preferences.